Translation/Helper/DomainHelper.php

```diff
@@ -88,7 +88,7 @@ public static function buildModuleDomainFromLegacySource($moduleName, $sourceFil
                 $source = substr($source, 0, -10);
             }
 
-            $source = ucfirst(strtr($source, ['_' => '']));
+            $source = ucfirst($source);
```

Comment:
So we keep underscores! Maybe we could camelize?

Reply:
No, we only keep them for the third part. If we camelized it then it would become a fourth part.

eg.
- `FooBarBar_test` => `Foo.Bar.Bar_test`
- `FooBarBarTest` => `Foo.Bar.Bar.Test`

Reply:
Right!